pyproject.toml

```diff
@@ -24,6 +24,7 @@ cognite-logger = "^0.6"
 questionary = "^2.0.1"
 mixpanel = "^4.10.1"
 packaging = "^24.1"
+typing-extensions = "^4.0"
```

Comment:
This is required by the cognite-sdk, so does not put any extra dependencies for us.